### ISSUE TITLE
How to set alarms for application on Operate First?

### ISSUE THREAD

**Issue by pacospace** [operate-first/support#78]
As Thoth Maintainer and Manager,

I would like to receive alarms on GitHub.

Reference: https://github.com/thoth-station/thoth-application/issues/854

**Comment by harshad16** [comment]
I think this issue is same as https://github.com/operate-first/apps/issues/108
and @hemajv had responded here: https://github.com/operate-first/SRE/tree/master/incident-management
do we want to make this a priority? @pacospace

**Comment by pacospace** [comment]
> I think this issue is same as [operate-first/apps#108](https://github.com/operate-first/apps/issues/108)
> and @hemajv had responded here: https://github.com/operate-first/SRE/tree/master/incident-management
> do we want to make this a priority? @pacospace

Thanks @harshad16 I just saw the link, then we can close this issue actually. And it would be nice to have before we start using MOC for advises.
